x-pack/legacy/plugins/siem/public/lib/compose/__mocks__/kibana_core.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createUiNewPlatformMock } from 'ui/new_platform/__mocks__/helpers';
+
+const npStart = createUiNewPlatformMock().npStart;
+
+export function useKibanaCore() {
```

Comment:
Thanks for providing mocks for both `useKibanaCore` & `useKibanaPlugins` hooks as part of this.